www-cronrobot/Gemfile

```diff
@@ -64,7 +64,7 @@ group :development do
 end
```

Comment:
would be better i think to move all create button on the right, y-aligned with the title:

![image](https://user-images.githubusercontent.com/8007722/132592674-c372f357-751c-46bd-aba7-af13a2e66ea6.png)

same in resources, notifications